openpype/hosts/blender/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -33,20 +34,21 @@ def has_uvs(obj: bpy.types.Object) -> bool:
     @classmethod
     def get_invalid(cls, instance) -> List:
         invalid = []
-        # TODO (jasper): only check objects in the collection that will be published?
-        for obj in [
-            obj for obj in instance]:
+        for obj in set(instance):
```

Comment:
When would the `instance` not have unique entries? I'd say maybe we should solve that instead of making use of `set` here? I suspect this could be slower for many instances with many nodes?

Reply:
Yes It will be absolutely true with #3463
Maybe not totaly true with this : [collect_instances.py : line 99](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/blender/plugins/publish/collect_instances.py#L99)
But we can simplify the loops without `set()`

Reply:
I'd say update the Collector to make to do `list(set(members))` once so that all validators, etc. benefit from it. But I'll leave it up to you whether that's sensible in the context of blender.